src/python/turicreate/test/test_regression.py

```diff
@@ -10,6 +10,10 @@
 import turicreate as tc
 import numpy as np
 
+import pytest
+
+pytestmark = [pytest.mark.minimal]
```

Comment:
Same as previous comment.

Reply:
that pr is merged into master. Let me merge master into it and push again.